testsuite/features/secondary/buildhost_osimage_build_image.feature

```diff
@@ -37,9 +37,8 @@ Feature: Build OS images
   Scenario: Check the OS image built as Kiwi image administrator
     Given I am on the Systems overview page of this "build_host"
     Then I should see a "[OS Image Build Host]" text
-    When I wait at most 3300 seconds until event "Image Build suse_os_image scheduled by kiwikiwi" is completed
-    And I wait at most 300 seconds until event "Image Inspect 1//suse_os_image:latest scheduled by kiwikiwi" is completed
-    And I navigate to "os-images/1/" page
+    When I wait until the image build is completed
+    And I am on the new image page
```

Comment:
This PR has two different topics imho.
I like the idea of hiding some navigation steps, to reduce lines in our scenarios, as far they are clear, as in this PR. So, from that topic LGTM.
But then, I would prefer to stick with the previous step definitions of the selected lines, I don't care about the "timeouts", fine to hide this part. I'm also fine to hide the user who scheduled the event. But I think we should still continue making it parametrizable and visible to the user the "suse_os_image" text, so the image name.

Reply:
Good idea, that way we can finally get the second case in that switch.